tests/selenium/test_switch_app.py

```diff
@@ -8,4 +8,9 @@ def test_switch_app(base_url, selenium, test_repository):
     page = page.switch_to_perfherder()
     assert page.header.active_app == 'Perfherder'
     page = page.switch_to_treeherder()
+    # Be aware that when switching back from Perfherder, it will try to
```

Comment:
I will write an explicit test for this case of an invalid repo, but in a separate PR.  https://bugzilla.mozilla.org/show_bug.cgi?id=1484032